src/QirRuntime/lib/QIR/bridge-qis.ll

```diff
@@ -13,7 +13,7 @@
 %Range = type { i64, i64, i64 }
 %Result = type opaque
 %String = type opaque
-%Pauli = type {i2}
+%Pauli = type i2

```

Comment:
Nit: Notice, that generated QIR doesn't actually use this type but uses i2 directly. My choice of not introducing type aliases was to match the generated code verbatim, in hope that it will be easier to find bugs like the wrong parameter in string creation. At some point I even had a script that attempted to automatically extract declared signatures from generated QIR and compare them to defined here but I couldn't get it reliable enough...

Reply:
OK, I'll keep that in mind.

I'll admit that now the types `i2` and `%Pauli` are the same (`%Pauli` being an alias/typedef of `i2`). Before this change, the were, strictly speaking, _different types_ (`%Pauli` was a struct with one field of type `i2`), and casting to `i32` was causing compilation error.

Reply:
This is probably how %Pauli was defined in the earlier versions of generated QIR when I copied it over into the bridge. But because it was never actually used, the outdated type didn't come to my attention.